addons/mentions/addon.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Highlight mentions",
+  "description": "Highlight when you are mentioned (e.g. @username).",
+  "tags": ["community", "profiles", "comments", "forums"],
+  "versionAdded": "1.34.0",
+  "customCssVariables": [
+    {
+      "name": "button-text",
+      "value": {
+        "type": "textColor",
+        "black": "#000",
+        "white": "#fff",
+        "source": {
+          "type": "settingValue",
+          "settingId": "color"
+        }
+      }
+    }
+  ],
+  "userscripts": [
+    {
+      "matches": ["projects", "studios", "profiles", "forums"],
+      "url": "userscript.js"
+    }
+  ],
+  "userstyles": [
+    {
+      "matches": ["projects", "studios", "profiles", "forums"],
+      "url": "userstyle.css"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Highlight color",
+      "id": "color",
+      "type": "color",
+      "default": "#855cd6"
+    },
+    {
+      "name": "Highlight direct links",
```

Comment:
Might want to attempt to clarify this setting. I didn't know what it did until I enabled it on a forum post, and I'm still not 100% sure what it does.

Making a concise name for this setting is hard. #5320 could help here.

Reply:
It matches `https://scratch.mit.edu/users/mybearworld` links as opposed to just `/users/mybearworld` links. This should probably just be a "highlight in forums" option.

Reply:
Maybe
```suggestion
      "name": "Highlight full links",
```